css-two-column-layout/styles.css

```diff
@@ -0,0 +1,65 @@
+.container {
+  max-width: 800px;
+  min-height: 70vh;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.row {
+  display: flex;
+  padding-left: 15px;
+  padding-right: 15px;
+  padding-bottom: 15px;
+}
+
+.col-full {
+  text-align: center;
+  margin-left: 100px;
+  margin-right: 100px;
+}
+
+.col-50 {
+  flex-basis: calc(100% / 2);
+}
+
+.col-50 > .row {
+  display: block;
```

Comment:
I'd avoid this pattern - the `.row` should be consistently set to `display: flex` throughout the entire stylesheet, otherwise you'll end up wondering why it's different in some places.